### ISSUE TITLE
Can not find `config.h` file when make in OSX

### ISSUE THREAD

**Issue by ppn029012** [HIT-SCIR/ltp#193]
[ 37%] Building CXX object src/xml4nlp/CMakeFiles/xml4nlp.dir/Xml4nlp.cpp.o
In file included from /Users/ppn029012/gitDir/myNLP/ltp-3.3.1/src/xml4nlp/Xml4nlp.cpp:12:
/Users/ppn029012/gitDir/myNLP/ltp-3.3.1/src/utils/strutils.hpp:19:17: warning: 
      comparison of unsigned expression >= 0 is always true
      [-Wtautological-compare]
  while (len -1 >=0 && (str[len-1] == ' '
         ~~~~~~ ^ ~
1 warning generated.
[ 38%] Linking CXX static library ../../lib/libxml4nlp.a
[ 38%] Built target xml4nlp
[ 38%] Building CXX object src/splitsnt/CMakeFiles/splitsnt_shared_lib.dir/SplitSentence.cpp.o
In file included from /Users/ppn029012/gitDir/myNLP/ltp-3.3.1/src/splitsnt/SplitSentence.cpp:2:
/Users/ppn029012/gitDir/myNLP/ltp-3.3.1/src/utils/strutils.hpp:19:17: warning: 
      comparison of unsigned expression >= 0 is always true
      [-Wtautological-compare]
  while (len -1 >=0 && (str[len-1] == ' '
         ~~~~~~ ^ ~
1 warning generated.
[ 39%] Linking CXX shared library ../../lib/libsplitsnt.dylib
[ 39%] Built target splitsnt_shared_lib
[ 39%] Building CXX object src/splitsnt/CMakeFiles/splitsnt_static_lib.dir/SplitSentence.cpp.o
In file included from /Users/ppn029012/gitDir/myNLP/ltp-3.3.1/src/splitsnt/SplitSentence.cpp:2:
/Users/ppn029012/gitDir/myNLP/ltp-3.3.1/src/utils/strutils.hpp:19:17: warning: 
      comparison of unsigned expression >= 0 is always true
      [-Wtautological-compare]
  while (len -1 >=0 && (str[len-1] == ' '
         ~~~~~~ ^ ~
1 warning generated.
[ 40%] Linking CXX static library ../../lib/libsplitsnt.a
[ 40%] Built target splitsnt_static_lib
[ 41%] Building CXX object src/segmentor/CMakeFiles/otcws.dir/otcws.cpp.o
/Users/ppn029012/gitDir/myNLP/ltp-3.3.1/src/segmentor/otcws.cpp:2:10: fatal error: 
      'config.h' file not found
#include "config.h"
         ^
1 error generated.
make[2]: *** [src/segmentor/CMakeFiles/otcws.dir/otcws.cpp.o] Error 1
make[1]: *** [src/segmentor/CMakeFiles/otcws.dir/all] Error 2
make: *** [all] Error 2


MacOSX 10.11.4
GNU Make 3.81
cmake 3.3.2

**Comment by endyul** [comment]
does the error still appear in ltp v3.3.2 ?

**Comment by ppn029012** [comment]
fixed in 3.3.2
